app/src/main/kotlin/com/mapzen/erasermap/model/LostClientManager.kt

```diff
@@ -44,7 +44,7 @@ class LostClientManager(val application: Application, locationFactory: LocationF
   }
 
   override fun connect() {
-    if (state == State.CONNECTING) {
+    if (state == State.CONNECTING || state == State.CONNECTED) {
```

Comment:
Looks like this change is not needed with https://github.com/mapzen/eraser-map/pull/750 merged